### PR TITLE
#22 Gradle Cache Fix

### LINKED ISSUE #22: Gradle Cache Issue
Интегрировать в приложение данный модуль: https://github.com/gradle/android-cache-fix-gradle-plugin

Он исправит проблемы с совместимостями сборки и CI, а также ускорит сборку проекта при локальной сборке.

Необходимо добавить зависимости в каждый модуль проекта!

### PR DESCRIPTION
**Pull request by PavelStalone**

### CODE REVIEW

**Comment by PavelStalone** [comment]
PR is here #22

**Review by PoroshinGA — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -4,6 +4,8 @@ plugins {
     id 'com.google.dagger.hilt.android'
     id 'androidx.navigation.safeargs'
     id 'de.undercouch.download'
+
+    id "org.gradle.android.cache-fix" version "2.8.0"
```
Comment (jacksever) [inline app/build.gradle:8]: Nope! Let's transfer this to `build.gradle.kts` for project module, not app module. And in app module you can add line as above

File: core/build.gradle [hunk 2]
```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'com.android.library'
+    id "org.gradle.android.cache-fix" version "2.8.0"
```
Comment (jacksever) [inline core/build.gradle:3]: No. Please see a comment above

File: feature/build.gradle [hunk 3]
```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'com.android.library'
+    id "org.gradle.android.cache-fix" version "2.8.0"
```
Comment (jacksever) [inline feature/build.gradle:3]: Remove version!

File: app/build.gradle [hunk 4]
```diff
@@ -41,6 +43,8 @@ android {
     }
 }
 
+apply plugin: "org.gradle.android.cache-fix"
```
Comment (jacksever) [inline app/build.gradle:46]: Remove

File: app/build.gradle [hunk 5]
```diff
@@ -4,6 +4,8 @@ plugins {
     id 'com.google.dagger.hilt.android'
     id 'androidx.navigation.safeargs'
     id 'de.undercouch.download'
+
```
Comment (jacksever) [inline app/build.gradle:7]: Extra line

**Review by jacksever — APPROVED** [review]

**Pull request merged**